Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -67,29 +70,29 @@ void copyLogs(const Mantid::DataHandling::EventWorkspaceCollection_sptr &from,
     }
   }
 }
-}
+} // namespace
 
 //==============================================================================================
 // Class LoadBankFromDiskTask
 //==============================================================================================
 /** This task does the disk IO from loading the NXS file,
-* and so will be on a disk IO mutex */
+ * and so will be on a disk IO mutex */
 class LoadBankFromDiskTask : public Task {
 
 public:
   //---------------------------------------------------------------------------------------------------
   /** Constructor
-  *
-  * @param alg :: Handle to the main algorithm
-  * @param entry_name :: The pathname of the bank to load
-  * @param entry_type :: The classtype of the entry to load
-  * @param numEvents :: The number of events in the bank.
-  * @param oldNeXusFileNames :: Identify if file is of old variety.
-  * @param prog :: an optional Progress object
-  * @param ioMutex :: a mutex shared for all Disk I-O tasks
-  * @param scheduler :: the ThreadScheduler that runs this task.
-  * @param framePeriodNumbers :: Period numbers corresponding to each frame
-  */
```

Comment:
Same issue as before: Please undo doxygen reformatting througout file to avoid future merge conflicts & hard to read diffs.